config/owasp/suppressions.xml

```diff
@@ -31,4 +31,11 @@
     <cve>CVE-2016-5394</cve>
     <cve>CVE-2016-6798</cve>
   </suppress>
+  <suppress  until="2020-06-01">
```

Comment:
although this should be set not more than a month but can be changed later as this Pr needs to go